modules/authentication/cognito/main.tf

```diff
@@ -71,23 +71,29 @@ module "aws_cognito_user_pool" {
     }
   ]
 
-  schemas = [
-    {
-      attribute_data_type      = "Boolean"
-      developer_only_attribute = false
-      mutable                  = true
-      name                     = "available"
-      required                 = false
-    },
-    {
-      attribute_data_type      = "Boolean"
-      developer_only_attribute = true
-      mutable                  = true
-      name                     = "registered"
-      required                 = false
-    }
-  ]
+  schemas = concat(
```

Comment:
Looks very good! 
You can remove the previous attributes they are not relevant.